src/coreclr/jit/jiteh.cpp

```diff
@@ -1868,7 +1868,7 @@ EHblkDsc* Compiler::fgTryAddEHTableEntries(unsigned XTnum, unsigned count, bool
         // yet, such as when we add an EH region for synchronized methods that don't already have one,
         // we start at zero, so we need to make sure the new table has at least one entry.
         //
-        unsigned newHndBBtabAllocCount = max(1u, compHndBBtabAllocCount + newCount);
+        unsigned newHndBBtabAllocCount = max(1u, newCount);
```

Comment:
`newCount` already includes `compHndBBtabAllocCount`, so this count was slightly overestimated

Reply:
I think it was like this before my recent changes, and I just left it this way.